apps/dashboard/config/current_user_singleton.rb

```diff
@@ -0,0 +1,11 @@
+class CurrentUserSingleton
+  def groups
+    @groups ||= OodSupport::User.new.groups.sort_by(&:id).tap { |groups|
+      groups.unshift(groups.delete(OodSupport::Process.group))
+    }.map(&:name).grep(/^P./) 
```

Comment:
This is OSC specific. We will want to change this to be more generic. I think the idea was to have groups return an array of strings that was the names of all the groups a user is in. So just what you had in the previous PR something like

```
def groups
  OodSupport::Process.groups.map(&:name)
end
```